tools/vscode-extension/src/config.ts

```diff
@@ -1,59 +1,87 @@
 import {commands, window, workspace} from "vscode";
-import {shortAddress} from "./address";
+import { addAddressPrefix } from "./address";
 
-export const ROOT_ADDR: string = shortAddress("0000000000000000000000000000000000000001");
+export const SERVICE_ADDR: string = "f8d6e0586b0a20c7";
```

Comment:
Will this be the same for the emulator and the mainnet?

Reply:
No, mainnet will have a different service address 😢 

I wasn't sure how to best handle this. Do we have any users using this extension with anything other than the emulator? Is that even possible at this point?

I can see _direct-to-mainnet_ or _direct-to-testnet_ being a feature in the future, but until we're ready to figure out how that works (i.e. with keys, accounts, etc), I figured it'd be safe to hardcode the emulator address for now.

Reply:
I was just curious, this is totally fine 👍 

I think the extension is mainly used for development